developing-with-nats-streaming/streaming.md

```diff
@@ -97,7 +97,7 @@ NATS Streaming supports several types of subscriptions:
 * Queue
 * Durable/Queue
 
-Regular subscriptions pick the location of their channel position on creation and it is stored while the subscriber is active. Durable subscriptions store their position in the streaming server. Queue subscriptions share a channel position. Durable/Queue subscriptions share a channel position stored in the server. All subscriptions can be configured with a starting position, but only new durable subscriptions and new regular subscriptions respect the request.
+Regular subscriptions pick the location of their channel position on creation and it is stored in the subscriber while the subscriber is active. Durable subscriptions store their position in the streaming server. Queue subscriptions share a channel position. Durable/Queue subscriptions share a channel position stored in the server. All subscriptions can be configured with a starting position, but only new durable subscriptions and new regular subscriptions respect the request.
```

Comment:
I would actually completely remove this paragraph. The one below states the same ideas, but better.